addons/account/models/company.py

```diff
@@ -11,6 +11,7 @@
 from odoo.tools.mail import is_html_empty
 from odoo.tools.misc import format_date
 from odoo.addons.account.models.account_move import MAX_HASH_VERSION
+from odoo.addons.base_vat.models.res_partner import _ref_vat
```

Comment:
Not sure why this didn't get reported yet, but `base_vat` has a dependency on `account`, not the other way around :skull:

Reply:
Oops yes. I didn't realize that. I guess we should have done these modifications in `base_vat` rather than `account`. Do you think it's okay if we do a new PR in 18.0 to move everything to `base_vat`, except keep the definitions of new fields in `account` (to avoid breaking stable), and add a migration script in master to move the fields from account to base_vat?

Reply:
(by move everything I meant move the computes and the views)

Reply:
Easy solution in stable is to manage the `ImportError` and defined an empty dict otherwise, or to move the definition of the dict in `account` directly and import it from there in `base_vat`

Reply:
will move the dict to `account`: https://github.com/odoo/odoo/pull/185059